### PR TITLE
Rename agora.common.crypto to agora.crypto

### PR DESCRIPTION
**Pull request by Geod24**
Make the package name consistent, potentially enabling 'package' access if needed in the future.
Also clean up the imports, including some other imports spotted along the way.

### CODE REVIEW

File: source/agora/node/TransactionPool.d [hunk 1]
```diff
@@ -610,7 +610,7 @@ unittest
 unittest
 {
     import agora.common.Amount;
-    import agora.common.crypto.Key;
+    import agora.crypto.Key;
```
Comment (kchulj) [inline source/agora/node/TransactionPool.d]: These 2 modules are imported in 5 unittests. Can we use a global `import`?

Comment (Geod24) [reply source/agora/node/TransactionPool.d]: Fixed

**Comment by Geod24** [comment]
Rebased

**Review by kchulj — APPROVED** [review]
LGTM.

**Review by linked0 — APPROVED** [review]

**Pull request merged**